streamdeck-plugin/src/event_handlers/emoji_react_event_handler.py

```diff
@@ -0,0 +1,37 @@
+from event_handlers.base_event_handler import EventHandler
+
+
+class EmojiReactEventHandler(EventHandler):
+    """
+    A reusable handler for Emoji reactions.
+    """
+
+    ACTION_TO_EMOJI = {
+        "com.chrisregado.googlemeet.emojireact.sparklingheart": "💖",
+        "com.chrisregado.googlemeet.emojireact.thumbsup": "👍",
+        "com.chrisregado.googlemeet.emojireact.partypopper": "🎉",
+        "com.chrisregado.googlemeet.emojireact.clappinghands": "👏",
+        "com.chrisregado.googlemeet.emojireact.facewithtearsofjoy": "😂",
+        "com.chrisregado.googlemeet.emojireact.facewithopenmouth": "😮",
+        "com.chrisregado.googlemeet.emojireact.cryingface": "😢",
+        "com.chrisregado.googlemeet.emojireact.thinkingface": "🤔",
+        "com.chrisregado.googlemeet.emojireact.thumbsdown": "👎"
+    }
+
+    STREAM_DECK_ACTION_PREFIX = "com.chrisregado.googlemeet.emojireact."
+
+    def _get_emoji_char_for_event(self, event: dict):
+        action = event['action']
+        emoji_char = self.ACTION_TO_EMOJI.get(action)
+        if not emoji_char:
+            raise NotImplementedError(f"The action '{action}' was requested, but there is no corresponding emoji.")
```

Comment:
Have we tested out what happens if this exception gets thrown? It might cause the plugin to crash, which would cause other buttons to stop working while the Stream Deck desktop app silently restarts the plugin the background. Since the Stream Deck SDK limits our ability to communicate with users, our best option might be to catch the exception in `_key_up_handler` and log it. Something like:

```
try:
  emoji_char = self._get_emoji_char_for_event(event)
except:
  self._logger.exception("Failed to find emoji for event!")
  return
```